src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,26 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DEFAULT_SIZE_OF_PASSWORD_AND_LOGIN = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null
+                || user.getLogin().length() < DEFAULT_SIZE_OF_PASSWORD_AND_LOGIN) {
+            throw new OwnRuntimeException("Login must be longer then 6 symbol");
```

Comment:
don't hardcode the length limit in the exception message, use your constant. Fix everywhere